otherlibs/unix/unix.mli

```diff
@@ -879,7 +879,9 @@ val open_process_args_in : string -> string array -> in_channel
    command to run, and the second argument specifies the argument array passed
    to the command.  This function runs the command in parallel with the program.
    The standard output of the command is redirected to a pipe, which can be read
-   via the returned input channel.
+   via the returned input channel. The program is looked up in the [PATH] (this
```

Comment:
Could you break a line before this new sentence, so that the behavior changes catches the eye?

Reply:
Done.